src/modules/map/components/LocationPopup.vue

```diff
@@ -98,6 +101,13 @@ watch(currentLang, () => {
 })
 watch(() => route.query, updateShareLink)
```

Comment:
Same here it seems that the share link is updated each time the query parameter is updated, should be done only it is displayed.